src/lib.rs

```diff
@@ -35,6 +35,7 @@ use std::marker::PhantomData;
 use std::mem;
 
 use rand::Rng;
+#[cfg(feature = "serde")]
```

Comment:
Shouldn't the feature be called `"serde_support"`?

Reply:
Yes, and I'm not sure how it worked at all.